### ISSUE TITLE
Creating (template for) Wiki in GitHub

### ISSUE THREAD

**Issue by deepdigger** [osmandapp/Osmand#8096]
Hello together,
I've been working the last days into getting started with developing in OsmAnd like modifing the routing algorithm, changing maps, starting testing a plugin with aidl, simulating the routing with the developement plugin...

However, this took me a really long time. Despite having some "Help-me" pages on the OsmAnd homepage it was a lot researching and finding different links or simply testing how the OsmAnd-App works.

I believe that there should be a better way to store important information than google groups, github issues or different (not really linked) parts of the OsmAnd Website.
That's why I want to ask you if you can create a github wiki page, where users can add and modify information. I'm thinking about important links, how to's, introductions...
I hope that this wiki could be run by users from outside the developement team, the only important thing is that it has a predefined structure. This would make working with OsmAnd so much easier because the developers from outside won't have to work everything out by their own, but could rely on a (huge) amount of structured information.

**Comment by vshcherb** [comment]
Agree, we need to have open-user space where everybody could edit cause website is pretty limited option as I see today.
